pressbooks/pressbooks-multi-institution
# feat: institution form rework

Issue #61 

This PR reworks the institution form following the requirements of #61.
It updates the database schema to reflect the new fields. It removes the user limit and adds additional fields to the form. In addition, it conditionally renders some of the fields based on the user role.

The institution list has also been updated to include new required information.

**How to test**

1. Before checking this branch out, deactivate the plugin so tables are deleted.
2. Check this branch out and activate the plugin back. Tables should be created and fields updated (we need to work on a migration process that is saner in the future)
3. Follow the requirements in the ticket 

> [!NOTE]
> The number of institutional managers is limited to a maximum of 3 per institution. The current multiselect component does not support limiting the number of selected elements (see https://github.com/greatislander/pressbooks-multiselect/issues/116 for additional details). For now this PR allows users to select more than 3 users but will only aissng the first 3 in the list.
